CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## [unreleased]
+### Fixed
+- Fixed an issue where the jobs and servers table in Traffic Portal would not clear a columns filter when its hidden
```

Comment:
s/columns/column's/ - possessive, not plural
s/its/it's/ - contraction, not possessive

Reply:
I am a great englisher.